include/zephyr/secure_storage/its.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2024
```

Comment:
This is confusing for me. Is this API supposed to be used directly or users should be using `psa_its_*` instead ?

AFAIU it is supposed to be the `psa`, so in this case why this header is public ? If it is not, shouldn't then it be agnostic and not use PSA types ? `psa_status_t`